### ISSUE TITLE
Widget visual issue with different currencies

### ISSUE THREAD

**Issue by ThomAce** [YiamiYo/Cura-PowerCostPlugin#6]
Different currencies might not showing up properly in widget.

In many countries, the currency name / short name is being put after the currency value.

I recomend to change the width of the widget.
Changing the default value to 60 or 75 gives better viewing.

Line 24: implicitWidth: iconSize + 60

Also changing the line 25 gives some more readability. Especially where the currency has no smaller fragments. This could be fixed either in the widget qml file or in the python code. I let you decide. I prefer using the below formula for writing out the value independently from currency.

text:  "%2 %1".arg(UM.Preferences.getValue("cura/currency")).arg(manager.printPowerCost.toFixed(2))

**Comment by YiamiYo** [comment]
Is this still a problem with the latest version? (v0.1.3)

**Comment by ThomAce** [comment]
Yes, as it is not giving implicit width in configuration for Cura Icon With Text.

In line 24, it is still the same as before. I can consider this just a wish for using it as country based version. (I mean, my modifications)

**In 0.1.3:**
		text: "%1 %2".arg(UM.Preferences.getValue("cura/currency")).arg(manager.printPowerCost.toFixed(2))

**My version:**
		implicitWidth: iconSize + 60
		text: "%2 %1".arg(UM.Preferences.getValue("cura/currency")).arg(manager.printPowerCost.toFixed(2))
